pallets/gear/src/benchmarking/utils.rs

```diff
@@ -138,6 +138,7 @@ where
     let ext_manager = ExtManager::<T>::default();
     let bn: u64 = Gear::<T>::block_number().unique_saturated_into();
     let root_message_id = MessageId::from(bn);
+    const DEFAULT_BLOCK_NUMBER: u32 = 0;
```

Comment:
const definition inside the fns are bad manners

Reply:
done